tests/lib/__init__.py

```diff
@@ -775,6 +775,11 @@ def need_bzr(fn):
         'Bazaar', ('bzr', 'version', '--short')
     )(fn))
 
+def need_svn(fn):
+    return pytest.mark.svn(need_executable(
+        'Subversion', ('svn', 'version')
```

Comment:
Should be using:
```python
def need_svn(fn):
    return pytest.mark.svn(need_executable(
        'Subversion', ('svn', '--version', '--quiet')
    )(fn))
```

Reply:
@benoit-pierre As an aside, GitHub now supports using `suggestion`.

```suggestion
        'Subversion', ('svn', '--version', '--quiet')
```